minethd.cpp

```diff
@@ -253,7 +253,7 @@ void minethd::work_main()
 	if(cuda_get_deviceinfo(&ctx) != 1 || cryptonight_extra_cpu_init(&ctx) != 1)
 	{
 		printer::inst()->print_msg(L0, "Setup failed for GPU %d. Exitting.\n", (int)iThreadNo);
-		std::exit(0);
+		exit(0);
```

Comment:
Ok, that compiles. Now, should I leave it where it is or move bellow `#if defined(__APPLE__)`